### ISSUE TITLE
Some buttons are not clickable again on the new version of Flet.

### ISSUE THREAD

**Issue by Hazelmeow** [flet-dev/flet#1267]
**Everything is OK on 0.5.0.dev1252 but some buttons are not clickable (they only work when first clicked) on 0.5.0.dev1290 and 0.6.0.dev**

**Code example to reproduce the issue:**

```python
Ahh.. sorry... I can't write a simple example to show this issue..
Maybe watch the video below..?
```

[https://github.com/wangzm5773/FileSharing/blob/main/example.mp4](https://github.com/wangzm5773/FileSharing/blob/main/example.mp4)

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.5.0.dev1290
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page:
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: C:\Program Files\Python312\Lib\site-packages
Requires: flet-core, httpx, oauthlib, packaging, watchdog, websocket-client, websockets
Required-by:
```

**Operating system**:

Windows 11

**Additional environment details: -**

**Comment by FeodorFitsner** [comment]
Link to example is broken.

**Comment by FeodorFitsner** [comment]
The issue has been fixed and fix will be available in Flet 0.5.2.
You can try the fix right now by upgrading to pre-release version of Flet: `pip install flet --pre --upgrade`.
Let me know how it worked.
